packages/teams-js/src/private/copilot/view.ts

```diff
@@ -0,0 +1,52 @@
+import { callFunctionInHostAndHandleResponse } from '../../internal/communication';
+import { ensureInitialized } from '../../internal/internalAPIs';
+import { ResponseHandler } from '../../internal/responseHandler';
+import { ApiName, ApiVersionNumber, getApiVersionTag } from '../../internal/telemetry';
+import { runtime } from '../../public/runtime';
+
+const copilotTelemetryVersionNumber: ApiVersionNumber = ApiVersionNumber.V_2;
+
+/**
+ * @hidden
+ * @internal
+ * Limited to Microsoft-internal use
+ * @beta
+ * @returns boolean to represent whether copilot.view capability is supported
+ *
+ * @throws Error if {@linkcode app.initialize} has not successfully completed
+ */
+export function isSupported(): boolean {
+  return ensureInitialized(runtime) && !!runtime.supports.copilot?.view;
+}
+
+/**
+ * Closes the side panel that is hosting the copilot app.
+ *
+ * @returns { Promise<Record<sting, never>> } - empty object to indicate successful closure of the side panel
+ * @throws { SdkError } - Throws a SdkError if host SDK returns an error as a response to this call
+ *
+ * @hidden
+ * @beta
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export async function closeSidePanel(): Promise<Record<string, never>> {
```

Comment:
Why does this need a return type? Can we return Promise<void> and have the Promise resolve to indicate a successful call?